### PR TITLE
 Add example with link to oficial example of "Material-ui with Typescript"

### PR DESCRIPTION
**Pull request by henriqueholtz**
## Documentation / Examples

- Add example with link to oficial example of "Material-ui with Typescript"

I'm following this similar example folder [examples/with-material-ui](https://github.com/vercel/next.js/tree/canary/examples/with-material-ui)

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
We already have a Material UI example, let's just add a link to that README: https://github.com/vercel/next.js/tree/canary/examples/with-material-ui

**Comment by henriqueholtz** [comment]
> We already have a Material UI example, let's just add a link to that README: https://github.com/vercel/next.js/tree/canary/examples/with-material-ui

@balazsorban44  Thanks for this appointment.

I've update my branch, and apply your suggestion [into this commit](https://github.com/vercel/next.js/pull/37499/commits/e70be802c0ece9424394fd1dc6387fef8cc2f1dc)

Please let me know if it's ok from now. Thanks!

**Comment by balazsorban44** [comment]
Hi, it looks like you accidentally did a lot of changes to the core. These should be reverted, as the only file that needs to change is this one: https://github.com/vercel/next.js/blob/canary/examples/with-material-ui/README.md

If it's easier, you can close this PR and create a new one with only the necessary changes.

File: examples/with-material-ui/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# Material UI example
+# Material UI examples
 
-This example have been moved here: [mui/material-ui](https://github.com/mui/material-ui/tree/master/examples/nextjs)
+### 1 - [Oficial example - ( Javascript )](https://github.com/mui/material-ui/tree/master/examples/nextjs)
+
+### 2 - [Oficial example - ( Typescript )](https://github.com/mui/material-ui/tree/master/examples/nextjs-with-typescript)
```
Comment (balazsorban44) [inline examples/with-material-ui/README.md:5]:
```suggestion
These examples are maintained outside of the Next.js repository:

1. Official [TypeScript example](https://github.com/mui/material-ui/tree/master/examples/nextjs-with-typescript)
2. Official [JavaScript example](https://github.com/mui/material-ui/tree/master/examples/nextjs)
```

Let's recommend TypeScript first. :+1:

**Pull request closed without merging**